public/documents/fari-games/charge-srd.md

```diff
@@ -400,11 +457,19 @@ When a player calls for a flashback scene:
   - the GM makes a fortune roll to see what happens.
   - the player makes an action roll to see what happens.
 
-# The World | Extras
+# Extras | Extras
```

Comment:
Since the idea of this document is a skeleton for writing a Powered by Charge game, does it make sense to move setting and world forces (two bits that probably should always be filled out to bring the world to live) above the extras that might not even be included? 

If forces are introduced first, it makes it more straightforward to let us also flesh out the world extra more by encouraging the GM to take a moment to consider what world forces are doing.

Reply:
That totally makes sense to me